src/test/mir-opt/copy_propagation_arg.rs

```diff
@@ -1,39 +0,0 @@
-// Check that CopyPropagation does not propagate an assignment to a function argument
```

Comment:
Do we have tests for all of these situations for the dest-prop opt?

Reply:
I ported this test over, the results look correct